.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,104 @@
+name: E2E Tests
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+  
+jobs:
+  run-e2e:
+    name: Install Datalens and Run E2E Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: datalens-tech/datalens
+          path: datalens
+      - uses: actions/checkout@v4
+        with:
+          path: datalens-ui
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'npm'
+          cache-dependency-path: ./datalens-ui/package-lock.json
+
+      - name: Run Docker Compose
+        working-directory: ./datalens
+        run: docker compose up -d
```

Comment:
We should change code here. This code starts hardcoded datalens image from current docker-compose.yml:

```
  datalens:
    container_name: datalens-ui
    image: ghcr.io/datalens-tech/datalens-ui:0.795.0
```

We need to change this file like this:

```
  datalens:
    container_name: datalens-ui
    dockerfile: ./datalens-ui/Dockerfile
```